src/Microsoft.DotNet.ProjectModel/ProjectReaderSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class ProjectReaderSettings
+    {
+        public string VersionSuffix = null;
```

Comment:
I'm not using setters or getters. I don't see any good reason to add it except for adding extra characters in the code although I'm good either way